pycript/utils.py

```diff
@@ -88,6 +88,69 @@ def update_json_key_value(json_obj, selectedlang, decryptionpath,enc_dec ,select
 
     return json_obj
 
+def update_raw_value(parameter_value, selectedlang, encryptionpath, enc_dec, selected_request_response_inc_ex_ctype,listofparam):
+    # Check if selectedreq_incexctype is None
+    param_name = parameter_value.getName()
+    param_value = parameter_value.getValue()
+    
+    if selected_request_response_inc_ex_ctype is None:
+        # Process all parameters in the parameter_valueect
+        new_value = enc_dec(selectedlang, encryptionpath, param_value)
+
+    else:
+        # Process parameters based on selectedparamtertype
+        if selected_request_response_inc_ex_ctype == "Include Parameters":
```

Comment:
we can have separate if condition, or if elif else. Adding else with the same if else condition make is difficult to read and not frndly